web-services/query/src/main/java/datawave/webservice/query/logic/composite/CompositeUserOperations.java

```diff
@@ -0,0 +1,104 @@
+package datawave.webservice.query.logic.composite;
+
+import com.google.common.collect.Sets;
+import datawave.security.authorization.AuthorizationException;
+import datawave.security.authorization.DatawavePrincipal;
+import datawave.security.authorization.DatawaveUser;
+import datawave.security.authorization.SubjectIssuerDNPair;
+import datawave.user.AuthorizationsListBase;
+import datawave.security.authorization.RemoteUserOperations;
+import datawave.webservice.query.exception.QueryException;
+import datawave.webservice.query.exception.QueryExceptionType;
+import datawave.webservice.query.result.event.ResponseObjectFactory;
+import datawave.webservice.result.GenericResponse;
+
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+
+public class CompositeUserOperations implements RemoteUserOperations {
```

Comment:
Add a brief description, even if it's only a couple of sentences.

Reply:
done